arcgis-ios-sdk-samples/Layers/Identify raster cell/IdentifyRasterCell.storyboard

```diff
@@ -0,0 +1,69 @@
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <scenes>
+        <!--Identify Raster Cell View Controller-->
+        <scene sceneID="sCe-GI-egB">
+            <objects>
+                <viewController id="CXg-Uu-bew" customClass="IdentifyRasterCellViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="84r-Oy-4W5">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="cIA-Y7-Zgc" userLabel="Map View" customClass="AGSMapView">
+                                <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                                <color key="backgroundColor" white="1" alpha="1" colorSpace="calibratedWhite"/>
+                            </view>
+                        </subviews>
+                        <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                        <constraints>
+                            <constraint firstItem="cIA-Y7-Zgc" firstAttribute="trailing" secondItem="84r-Oy-4W5" secondAttribute="trailing" id="4YM-8P-n1E"/>
+                            <constraint firstItem="cIA-Y7-Zgc" firstAttribute="leading" secondItem="84r-Oy-4W5" secondAttribute="leading" id="KMx-67-KUl"/>
+                            <constraint firstItem="cIA-Y7-Zgc" firstAttribute="top" secondItem="84r-Oy-4W5" secondAttribute="top" id="QlH-1u-Gva"/>
+                            <constraint firstItem="cIA-Y7-Zgc" firstAttribute="bottom" secondItem="84r-Oy-4W5" secondAttribute="bottom" id="pr9-bs-jmq"/>
+                        </constraints>
+                        <viewLayoutGuide key="safeArea" id="cQH-id-UEf"/>
+                    </view>
+                    <connections>
+                        <outlet property="calloutStackView" destination="Wqr-jb-4el" id="iS2-bn-zE1"/>
+                        <outlet property="mapView" destination="cIA-Y7-Zgc" id="Hqg-Rn-lha"/>
+                    </connections>
+                </viewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="P5h-DC-Dtg" userLabel="First Responder" customClass="UIResponder" sceneMemberID="firstResponder"/>
+                <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" distribution="fillProportionally" spacing="10" translatesAutoresizingMaskIntoConstraints="NO" id="Wqr-jb-4el" customClass="IdentifyRasterCellStackView" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target">
```

Comment:
I think you can set the alignment to Leading so that the labels are only as wide as they need to be, instead of the full width of the stack view:

![image](https://user-images.githubusercontent.com/2257493/83201082-9d9f8700-a0f9-11ea-8d6e-91f5be3a3e2f.png)

Reply:
This is fixed.

In order to keep the width of callout a constant, a constraint was added to the stack view, so that when dragging on the map, the callout won't keep adjusting its width according to the width of the label (otherwise looks a bit glitchy 🤔).
 
I'm not sure about the best practice of setting up views in a stack view.  
Currently I set all the subview's height to 0, and only add a width constraint to the stack view, expecting autolayout will provide proper height for each view at runtime. However, the views does not display well in the xib because their heights are set to zero. Is this the correct approach for auto adjusting heights for the labels? Thanks.

Reply:
Sounds good. I'll revert this change.